api.py

```
@@ -573,172 +573,376 @@ async def bulk_update_user_xp(
         print(f"Error during bulk XP update: {e} — {safe_id}")
 
 
-def get_table_definitions() -> dict[str, str]:
-    """Return the table DDL definitions used by create_tables.
+def get_table_defs() -> dict[str, "TableDef"]:
```

Flake8 (7.3.0)

[error] undefined name 'TableDef'

(F821)